From: Bart Van Assche <bart.vanassche@sandisk.com>
Date: Fri, 23 Dec 2016 13:23:45 +0100
Subject: target: Remove se_tmr_req.tmr_lun
References: bsc#1036489
Git-commit: 4f4c6c3ddb15af396cd8cf81b487b59261199ee9
Patch-Mainline: v4.11-rc1

Member tmr_lun of se_tmr_req is set but not used. Hence remove it.

Signed-off-by: Bart Van Assche <bart.vanassche@sandisk.com>
Reviewed-by: Hannes Reinecke <hare@suse.com>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Reviewed-by: Himanshu Madhani <himanshu.madhani@cavium.com>
Cc: Giridhar Malavali <giridhar.malavali@cavium.com>
Signed-off-by: Nicholas Bellinger <nab@linux-iscsi.org>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/target/target_core_device.c | 1 -
 include/target/target_core_base.h   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/target/target_core_device.c b/drivers/target/target_core_device.c
index e4aaf6f..e0755e2 100644
--- a/drivers/target/target_core_device.c
+++ b/drivers/target/target_core_device.c
@@ -167,7 +167,6 @@ int transport_lookup_tmr_lun(struct se_cmd *se_cmd, u64 unpacked_lun)
 	rcu_read_lock();
 	deve = target_nacl_find_deve(nacl, unpacked_lun);
 	if (deve) {
-		se_tmr->tmr_lun = rcu_dereference(deve->se_lun);
 		se_cmd->se_lun = rcu_dereference(deve->se_lun);
 		se_lun = rcu_dereference(deve->se_lun);
 		se_cmd->pr_res_key = deve->pr_res_key;
diff --git a/include/target/target_core_base.h b/include/target/target_core_base.h
index 9ffcad2..2ffeec8 100644
--- a/include/target/target_core_base.h
+++ b/include/target/target_core_base.h
@@ -404,7 +404,6 @@ struct se_tmr_req {
 	void 			*fabric_tmr_ptr;
 	struct se_cmd		*task_cmd;
 	struct se_device	*tmr_dev;
-	struct se_lun		*tmr_lun;
 	struct list_head	tmr_list;
 };
 
-- 
1.8.5.6

